pages/api/webhooks/customer-created.ts

```diff
@@ -0,0 +1,43 @@
+import { NextApiRequest, NextApiResponse, NextApiHandler } from "next";
+import { SALEOR_DOMAIN_HEADER } from "../../../constants";
+import Klaviyo from "../../../lib/klaviyo";
+
+const expectedEvent = "customer_created";
+
+const handler: NextApiHandler = async (
+  request: NextApiRequest,
+  response: NextApiResponse
+) => {
+  const saleorDomain = request.headers[SALEOR_DOMAIN_HEADER];
```

Comment:
checking Saleor domain and received event should be middlewares and the should also be inherited from tempate (as rest of our application will heavely rely on them also.